tests/foreman/api/test_contentview.py

```diff
@@ -823,14 +822,13 @@ class TestContentViewRedHatContent:
 
     @pytest.mark.skip_if_not_set('fake_manifest')
```

Comment:
Should we remove this skip, since we aren't using `fake_manifest` now?

Reply:
The current plan is to leave `fake_manifest` in place as a backup, but yes, we should remove `fake_manifest` skips. I'll make that change.